app/src/main/java/com/peacecorps/pcsa/circle_of_trust/Trustees.java

```diff
@@ -43,21 +44,34 @@
 
     SharedPreferences sharedpreferences;
     SharedPreferences.Editor editor;
+    private Button okButton;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_trustees);
 
-        System.out.println(comradeEditText.size());
-        comradeEditText.add((EditText) findViewById(R.id.comrade1EditText));
```

Comment:
Explain the rationale behind your choice of removing these and breaking the functionality in to two.